### PR TITLE
Connect send donation button to Kovan FanProxy contract

### PR DESCRIPTION
**Pull request by janclarin**
- Uses FanProxy contract based on network id (e.g. from Metamask)
- Updates transaction modal etherscan link to use Kovan by default and open in a new tab
- Adds `addresses` package, copied from [create-eth-app](https://github.com/paulrberg/create-eth-app)
  - Contains addresses and abis for all common Aave tokens and contracts
  - Adds FanProxy address on Kovan

https://user-images.githubusercontent.com/2449384/107014622-b6644000-6750-11eb-99c3-c7bc6038bad9.mov

### CODE REVIEW

**Review by chailey — APPROVED** [review]

File: client/src/components/Fan.js [hunk 1]
```diff
@@ -46,10 +47,12 @@ class Fan extends Component {
     try {
       const web3 = new Web3(provider);
       const networkId = await web3.eth.net.getId();
-      const deployedNetwork = FanProxyContract.networks[networkId];
+      const fanProxyAddress = addresses[networkId]
```
Comment (priyankaranke) [inline client/src/components/Fan.js:50]: This is a lot cleaner than what we had before. Nice!

**Review by priyankaranke — APPROVED** [review]
Regarding `src/addresses/abis/`, all the abis are already available in `src/contracts/` right?

File: client/src/components/Fan.js (hunk 1 above)
Comment (janclarin) [reply client/src/components/Fan.js:50]: Thanks!

**Comment by janclarin** [comment]
> Regarding `src/addresses/abis/`, all the abis are already available in `src/contracts/` right?

`src/contracts` only contains the abis for contracts that are used as deps in our contracts. This has the aToken abi too, which will be useful for checking the balance.

**Pull request merged**